store/store.go

```diff
@@ -235,3 +248,23 @@ type Identity struct {
 	// this one.
 	Owner ProviderIdentity
 }
+
+// MFACredential stores data about a multi-factor credential.
+type MFACredential struct {
+	// Username contains the username of the user to which the
+	// multi-factor credential belongs.
+	Username string
```

Comment:
It would be better to use a field that is fixed for this, under some circumstances the username can change. ProviderID is probably  good choice.

Reply:
i proposed using identity.ID...

Reply:
The only thing about using identity.ID is that you will always have to look that up in the database, whereas ProviderID can normally be derived from information provided by the identity provider. In this case it probably doesn't matter.

Reply:
changed to ProviderID